EngineHacks/SkillSystem/Skills/TurnSkills/Installer.event

```diff
@@ -14,7 +14,10 @@ WORD HoardersBaneID
 VulneraryItemID_Link:
 WORD Vulnerary 
 
-
+DrizzleID_Link:
```

Comment:
Please don't use this syntax for function label names. `DrizzleID_Link` implies that the byte following this label will be the value of `DrizzleID`, not the function. Convention would name this label `Drizzle`, or if that symbol is already in use `DrizzleFunc`.

Reply:
I understand, Is it worth making the amends SnowWarning, since that's the last skill with this issue?

Reply:
yeah

Reply:
Ah sorry, I realize that comment looks dumb now. I thought you'd made the changes for some reason.